Formula/mkvtoolnix.rb

```diff
@@ -71,7 +71,7 @@ def install
                           "--with-docbook-xsl-root=#{Formula["docbook-xsl"].opt_prefix}/docbook-xsl",
                           "--with-extra-includes=#{extra_includes}",
                           "--with-extra-libs=#{extra_libs}",
-                          "--disable-qt"
+                          "--disable-gui"
```

Comment:
If `qt@5` is still a runtime dep as opposed to build-only, then there's no need to do `--disable-gui`, no?

Reply:
My thought was that we wanted to keep tool being command-line only. I'm open to build the GUI if needed.

Reply:
Avoiding the GUI when they introduce extra dependencies is usually the right thing to do. Now it seems like the GUI comes for free, so there doesn't seem to be a strong reason to disable it.

Reply:
Oops, that was a lie. We still do need extra dependencies to build the GUI. We can disable it again :)

Reply:
Yep: `configure: error: Could not find the cmark library`